the-collab-lab/tcl-17-smart-shopping-list
# Add comments with issue AC for preventing duplicate list items

## Description

This code will alert the user if an item already exists in the user's list.  

## Related Issue

Closes #7 

## Acceptance Criteria


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |

## Updates
**'tomato' is already in the list**

![Screen Shot 2021-01-19 at 5 21 57 PM](https://user-images.githubusercontent.com/14856004/105106098-0c5f9500-5a7b-11eb-9363-8d302efd3476.png)

**tomato spelled with special characters is recognized as 'tomato'**

![Screen Shot 2021-01-19 at 5 22 39 PM](https://user-images.githubusercontent.com/14856004/105106106-0f5a8580-5a7b-11eb-9b57-57143658e193.png)


## Testing Steps / QA Criteria

* One way to test the site is to click on the `details` button below to open the Netlify preview

![Screen Shot 2021-01-12 at 6 05 51 PM](https://user-images.githubusercontent.com/14856004/104389400-ff81f500-5500-11eb-95ae-4c0169a069cb.png)

* Otherwise from your terminal, pull down this branch with `git pull origin RR-VT-prevent-duplicate` and check that branch out with `git checkout RR-VT-prevent-duplicate`
* `npm start` to launch the app.
* From add item page enter an item that already exists to see the error message
* List view reflects exact user input
